homeassistant/components/overkiz/strings.json

```diff
@@ -28,6 +38,12 @@
           "username": "[%key:common::config_flow::data::username%]",
           "password": "[%key:common::config_flow::data::password%]",
           "verify_ssl": "[%key:common::config_flow::data::verify_ssl%]"
+        },
+        "data_description": {
+          "host": "The hostname or IP address of your Overkiz hub.",
+          "username": "The username of your cloud account (app).",
+          "password": "The password of your cloud account (app).",
+          "verify_ssl": "Verify the SSL certificate. Select this only if you are connecting via the hostname."
```

Comment:
Just a question: Should a valid SSL certificate (i.e. not self-signed) be mentioned here as well?
Just "connecting via the hostname" does not imply that an SSL connection will work, does it? Or do Overkiz hubs always have a valid SSL certificate?

Reply:
OverKiz hubs do have a self-signed SSL certificate and the underlying pyOverkiz package adds their root cert to the cert store. So if users connect via the hostname, the certificate can be verified.